### PR TITLE
Fixd a typo in Readme

### PR DESCRIPTION
**Pull request by sirozhahehe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Library that helps you to check if value equals 11.
 
 # Installation
 
-Hope you how to install packages.
+Hope you know how to install packages.
```
Comment (Snegirekk) [inline README.md:7]: I think it might be "guess" instead of "know"

Comment (sirozhahehe) [reply README.md:7]: No, I guess it is exactly "know". @Snegirekk

Comment (Snegirekk) [reply README.md:7]: Not sure..

Comment (sirozhahehe) [reply README.md:7]: I am sure

**Pull request closed without merging**